FLT/QuaternionAlgebra/NumberField.lean

```diff
@@ -193,27 +196,36 @@ theorem GL2.TameLevel.isOpen : IsOpen (GL2.TameLevel S).carrier :=
 theorem GL2.TameLevel.isCompact : IsCompact (GL2.TameLevel S).carrier :=
   sorry
 
+attribute [local instance] Algebra.TensorProduct.rightAlgebra in
 noncomputable def QuaternionAlgebra.TameLevel (r : Rigidification F D) :
-    Subgroup ((FiniteAdeleRing (𝓞 F) F) ⊗[F] D)ˣ :=
+    Subgroup (D ⊗[F] (FiniteAdeleRing (𝓞 F) F))ˣ :=
   Subgroup.comap (Units.map r.toMonoidHom) (GL2.TameLevel S)
 
-instance : TopologicalSpace ((FiniteAdeleRing (𝓞 F) F) ⊗[F] D) :=
+attribute [local instance] Algebra.TensorProduct.rightAlgebra in
+instance : TopologicalSpace (D ⊗[F] (FiniteAdeleRing (𝓞 F) F)) :=
   moduleTopology (FiniteAdeleRing (𝓞 F) F) _
 
-instance : IsModuleTopology (FiniteAdeleRing (𝓞 F) F) ((FiniteAdeleRing (𝓞 F) F) ⊗[F] D) :=
+attribute [local instance] Algebra.TensorProduct.rightAlgebra in
+instance : IsModuleTopology (FiniteAdeleRing (𝓞 F) F) (D ⊗[F] (FiniteAdeleRing (𝓞 F) F)) :=
   ⟨rfl⟩
 
-instance : IsTopologicalRing ((FiniteAdeleRing (𝓞 F) F) ⊗[F] D) :=
-  IsModuleTopology.isTopologicalRing (FiniteAdeleRing (𝓞 F) F) ((FiniteAdeleRing (𝓞 F) F) ⊗[F] D)
+attribute [local instance] Algebra.TensorProduct.rightAlgebra in
+instance : Module.Finite (FiniteAdeleRing (𝓞 F) F) (D ⊗[F] FiniteAdeleRing (𝓞 F) F) := sorry
```

Comment:
I'm pretty sure Ruben already did this!

Reply:
It's `instance _root_.Module.Finite.base_change_right` in `FLT/AutomorphicForm/QuaternionAlgebra/Defs.lean`